preview.geoshp/Dockerfile

```diff
@@ -9,14 +9,18 @@ ARG GITSHA1="unknown"
 COPY requirements.txt /home/clowder/
 
 # Install any programs needed
-#USER root
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && \
+       DEBIAN_FRONTEND=noninteractive apt-get install -y \
        file \
-       gdal-bin \
-       python-gdal \
-       p7zip-full \
-    && pip install --upgrade  -r /home/clowder/requirements.txt \
-    && rm -rf /var/lib/apt/lists/*
+       python3 \
+       python-is-python3 \
+       python3-pip \
+       python3-gdal \
+       python3-rtree \
+       python3-scipy \
+       p7zip-full && \
+       pip3 install -r /home/clowder/requirements.txt && \
```

Comment:
just as FYI, i like to do :

```
RUN install packages

COPY requirements.txt
RUN pip3 install -r requirements.txt
```

Advantage, if you just update requirements it does not break the cache until that line in the Dockerfile.